modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/AbstractPlannerTest.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.ignite.internal.processors.query.calcite.planner;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
Wrong import sort order (run `mvn initialize -Pcheckstyle` to check)

Reply:
Sorry, fixed.